src/ParseSchema.js

```diff
@@ -161,6 +165,21 @@ class ParseSchema {
     }
   }
 
+  /**
+   * Sets Class Level Permissions when creating / updating a Schema.
+   * EXERCISE CAUTION, running this may override CLP for this schema and cannot be reversed
+   *
+   * @param {Object} clp Class Level Permissions
+   * @return {Parse.Schema} Returns the schema, so you can chain this call.
+   */
+  setCLP(clp: { [key: string]: mixed }) {
+    if (!clp) {
```

Comment:
why return here.  shouldn't you be able to 'unset' by sending null?

Reply:
True